velox/functions/lib/string/StringImpl.h

```diff
@@ -187,7 +187,10 @@ stringPosition(const T& string, const T& subString, int64_t instance = 0) {
 
 /// Replace replaced with replacement in inputString and write results to
 /// outputString.
-template <typename TOutString, typename TInString>
+template <
+    bool ignoreEmptyReplaced = false,
```

Comment:
Is `ignoreEmptyReplaced` moving to the end of template argument more reasonable?

Reply:
Thanks for the comment. Done.